docs/install/installing-istio.md

```diff
@@ -22,71 +22,51 @@ You can view the latest tested Istio version on the [Knative Net Istio releases
-
-You can install Istio with or without a service mesh:
-
-- [Installing Istio without sidecar injection](#installing-istio-without-sidecar-injection) (Recommended
-     default installation)
-
-- [Installing Istio with sidecar injection](#installing-istio-with-sidecar-injection)
-
-If you want to get up and running with Knative quickly, we recommend installing
-Istio without automatic sidecar injection. This install is also recommended for
-users who don't need the Istio service mesh, or who want to enable the service
-mesh by [manually injecting the Istio sidecars][1].
-
-#### Installing Istio without sidecar injection
-
-Enter the following command to install Istio:
-
-To install Istio without sidecar injection:
+### Basic installation with istioctl
+You can easily customize your Istio installation with `istioctl`.
 
 ```sh
 istioctl install -y
 ```
 
-#### Installing Istio with sidecar injection
-
-If you want to enable the Istio service mesh, you must enable [automatic sidecar
-injection][2]. The Istio service mesh provides a few benefits:
+### Forming a service mesh
+The Istio service mesh provides a few benefits:
 
-- Allows you to turn on [mutual TLS][3], which secures service-to-service
+- Allows you to turn on [mutual TLS][1], which secures service-to-service
   traffic within the cluster.
 
-- Allows you to use the [Istio authorization policy][4], controlling the access
+- Allows you to use the [Istio authorization policy][2], controlling the access
   to each Knative service based on Istio service roles.
 
-For automatic sidecar injection, set `autoInject: enabled` in addition to the earlier
-operator configuration.
+If you want to use Istio as a service mesh, you must make sure that istio sidecars
+are injected to all `pods` that should be part of the service mesh. There are two ways to achieve this:
 
-```
-    global:
-      proxy:
-        autoInject: enabled
-```
+- Use [automatic sidecar injection][3] and set the `istio-injection=enabled` label on all `namespaces`
+  that should be part of the service-mesh
 
-#### Using Istio mTLS feature
+- Use [manual sidecar injection][4] on all `pods` that should be part of the service-mesh
 
+
+### Using Istio mTLS feature with Knative
 Since there are some networking communications between knative-serving namespace
 and the namespace where your services running on, you need additional
 preparations for mTLS enabled environment.
 
-1. Enable sidecar container on `knative-serving` system namespace.
+!!! note
+    It is strongly recommended to use [automatic sidecar injection][3]
+    to avoid manually injection sidecars to all `pods` in `knative-serving`.
+
+1. Enable sidecar injection on `knative-serving` system namespace.
 
     ```bash
     kubectl label namespace knative-serving istio-injection=enabled
     ```
 
-1. Set `PeerAuthentication` to `PERMISSIVE` on knative-serving system namespace
+2. Set `PeerAuthentication` to `PERMISSIVE` on knative-serving system namespace
```

Comment:
The number `1` should be fine as markdown automatically assign to the correct number.